entity-framework/core/extensions/index.md

```diff
@@ -45,6 +45,13 @@ CatFactory is a scaffolding engine for .NET Core and Entity Framework Core. The
 
 [GitHub repository](https://github.com/hherzl/CatFactory.EntityFrameworkCore)
 
+### LoreSoft's Entity Framework Core Generator
+
+Entity Framework Core Generator (efg) is a .NET Core CLI tool that can generate EF Core models from an existing database, much like `dotnet ef dbcontext scaffold`. However its different int that it also supports safe code [regeneration](https://efg.loresoft.com/en/latest/regeneration/). Regeneration is accomplished either via region replacement or by parsing mapping files. It also supports view model, validation and object mapper generation. For more information, see the tutorial and the product documentation.
```

Comment:
Minor typo,  `However its different int that it also supports safe code`...  should be `However its different in that it also supports safe code`.

Reply:
Good catch! It's also the wrong version of "its" :smile: